storage/gr716-flash/flashsrv.c

```diff
@@ -430,6 +440,16 @@ static void flashsrv_devThread(void *arg)
 				msg.o.err = EOK;
 				break;
 
+			case mtGetAttr:
+				if (msg.i.attr.type == atSize) {
+					msg.o.attr.val = memory->ctx.properties->totalSz;
+					msg.o.err = 0;
```

Comment:
nit: both implementations use `EOK` for success instead of `0`

Reply:
EOK is non-standard and it alas leaked from the kernel. I removed EOKs from this function instead